### ISSUE TITLE
--time option not recognised?

### ISSUE THREAD

**Issue by ruipacheco** [ogham/exa#8]
```
ruipacheco @ traitor in ~/Projects/Assorted/exa                                                                                                                                                       
$ target/release/exa -lhart ~/Projects/xxx/yyy/
Unrecognized option: '--time /Users/ruipacheco/Projects/xxx/yyy/'.
command failed with status: 3
```

I'm using the fish shell, if that matters.

**Comment by adouzzy** [comment]
An argument for -t is needed. for example "exa -lt mod"
BUT, "exa -lt mod" doesn't sort file by modified date. The output is the same as "exa -l".

**Comment by ogham** [comment]
@adouzzy is correct - you need to pass in an argument. It's interpreting `-lhart ~/path` as `-lhar --time=~/path`, then complaining that `~/path` isn't a valid time field option - which, of course, it isn't! I'm not sure if there's a genuine way to check whether this has happened, though, as the Rust getopts crate doesn't require an `=` when passing options to parameters.

There's currently no way to sort files by modification date, because, err... I completely forgot to do it. Give me a few minutes...

**Comment by ogham** [comment]
FYI, I just implemented sort-by-modified date in a4e17193d990340b163efceff8eb5e513199d704. Try `--sort=modified`.
